extensions/ql-vscode/src/extension.ts

```diff
@@ -771,9 +771,7 @@ async function activateWithInstalledDistribution(
         // Note we must update the query history view after showing results as the
         // display and sorting might depend on the number of results
       } catch (e) {
-        const err = asError(e);
-        err.message = `Error running query: ${err.message}`;
-        item.failureReason = err.message;
+        item.failureReason = `Error running query: ${getErrorMessage(e)}`;
```

Comment:
This changes behaviour for the thrown error since the error message now doesn't include `Error running query:`. The message from the error is probably shown to the user directly since this method is called by a command, so this might change user-facing behaviour as well (depending on if the thrown error is an instance of `Error`).

Reply:
Oh your right because `asError` can just return `e` instead of a new error, so actually this does edit the existing `e` which we later throw 🤯 

I'll revert this change and leave this file alone.